scheds/rust/scx_bpfland/src/bpf/main.bpf.c

```diff
@@ -429,6 +429,14 @@ static s32 pick_idle_cpu(struct task_struct *p, s32 prev_cpu, u64 wake_flags, bo
 	idle_smtmask = scx_bpf_get_idle_smtmask();
 	idle_cpumask = scx_bpf_get_idle_cpumask();
 
+	/*
```

Comment:
Might be able to save a tctx lookup if this is placed before line 418. This is a good pattern!